assets/js/src/core/modules/element/actions/rename/use-rename.tsx

```diff
@@ -47,11 +48,34 @@ export const useRename = (elementType: ElementType): UseRenameHookReturn => {
   }
 
   const renameContextMenuItem = (node: TreeNodeProps): ItemType => {
+    console.log('permission', node.permissions.rename)
```

Comment:
Please remove the console.log and the temporary comment

Reply:
done